desktop_version/src/editor.cpp

```diff
@@ -3280,6 +3285,31 @@ dmwidth(void)
     return 40;
 }
 
+int cycleThroughCustomResources(int current, std::map <int, std::vector<SDL_Surface*>>& Map)
+{
+	bool currentonefound = false;
+	int currentsheet = current;
+	int firstsheet = 0;
+	int nextsheet = 0;
+	for (auto sheet : Map) {
+		if (firstsheet == 0)
+			firstsheet = sheet.first;
+
+		if (currentonefound) {
+			nextsheet = sheet.first;
+			break;
+		}
+
+		if (sheet.first == currentsheet)
+			currentonefound = true;
+    }
+
+	if (!currentonefound)
+		nextsheet = firstsheet;
+
+	return nextsheet;
+}
+
```

Comment:
This function should be indented with spaces, since the surrounding code is also indented with spaces.

Reply:
Also why use camelCase for the function name? Most of the functions in `editor.cpp` are named using lowercase-underscores.

Reply:
I misspoke, most of the functions are actually named using lowercasenoseparators. But at least lowercase_underscores is more tolerable than camelCase.